generator/generate-one.sh

```diff
@@ -68,6 +68,9 @@ if [[ found_library_in_pom -eq 0 ]]; then
 else
   echo "adding module $client_lib_name to pom"
   sed -i "/^  <modules>/a\ \ \ \ <module>"$client_lib_name"</module>" pom.xml
+  # also write to generated/README.md
+  # format |name|distribution name|
+  echo -e "|$client_lib_name|$client_lib_groupid:$client_lib_artifactid|" >> README.md
```

Comment:
We'll probably end up with duplicates over time. Might have to add some shell script like `cat README.md | grep .*:.* | sort | uniq >> README.md`

Reply:
The `if` logic checks for if this module is already added to the `generated-parent` pom. I am assuming here that a new module == not in `generated-parent` pom == not in `README.md`.
But there is no harm adding this de-dup logic just in case.